### ISSUE TITLE
Subject: Compilation Error in LaMEM: Assistance Needed

### ISSUE THREAD

**Issue by yan1in** [UniMainzGeo/LaMEM#7]
I am currently working with LaMEM and encountered an issue while compiling using the "make mode=opt all" command. The error message I received is as follows:

```
In file paraViewOutBin.cpp, line 378:
Error: macro "SETERRQ" is passed 4 arguments but takes only 3
  378 |   SETERRQ(PETSC_COMM_WORLD, PETSC_ERR_USER, "Too many phase aggregates specified! Max allowed: %lld", (LLD)_max_num_phase_agg_);
```

Could you kindly provide guidance on how to resolve this compilation error? I appreciate your assistance in this matter.

**Comment by boriskaus** [comment]
which version of PETSc are you using?

**Comment by yan1in** [comment]
According to the document, install version 3.16.4 of PETSc

**Comment by yan1in** [comment]
Thank you very much for your assistance. The issue was resolved after I reinstalled PETSc version 3.18.6.

**Comment by boriskaus** [comment]
I've updated the documentation scripts, so now the docs are up-to-date again
